src/main/java/Duke/Tasks/Deadline.java

```diff
@@ -0,0 +1,78 @@
+package Duke.Tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents the Deadline class
```

Comment:
Would it be redundant to say that the Deadline class represents the Deadline class? I noticed the same thing in your other classes as well.